.gitignore

```diff
@@ -30,3 +32,4 @@
 /po/LINGUAS
 /test/common/
 /test/images/
+/test/static-code
```

Comment:
Why would this be part of gitignore? It's cockpit's custom code there which we might want to update in the future for the whole repo.

Reply:
Ah right, this should be mantained in cockpit's repo, and not c-machines. Then I agree that this can be gitignored.